code/game/objects/items/weapons/storage/boxes.dm

```diff
@@ -76,6 +76,16 @@
 	new /obj/item/weapon/reagent_containers/hypospray/medipen(src)
 	return
 
+/obj/item/weapon/storage/box/security
+
+/obj/item/weapon/storage/box/security/New()
+	..()
+	contents = list()
+	new /obj/item/clothing/mask/gas/sechailer(src)

```

Comment:
I dont care about appearance i would rather they be in the wardens office as a hand out. If you really want them to have a special mask recolour the standard mask red.

The sec hailing is a stupid gimmick that is there to encourage abuse. It didnt do what it was originally adcertised for help make people aware when they are wanted.